src/main/java/ro/unibuc/hello/data/Task.java

```diff
@@ -0,0 +1,25 @@
+package ro.unibuc.hello.data;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDateTime;
+
+@Document
+@Getter
+@Setter
+public class Task {
+
+	String id;
+	String projectId;
+	String type;
+	String summary;
+	String description;
+	String reporterId;
+	String assigneeId;
+	String priority;
```

Comment:
Shouldn't those fields use the Enum types instead of `String`?

Reply:
It depends, i thought that we shouldn't complicate with mapping errors for typing wrong on the post call (as we don't have ui with select options) but i'll change with Enum

Reply:
Looks like it works out-of-the-box on my machine, but fair enough.